portfolio/src/main/webapp/script.js

```diff
@@ -1,4 +1,63 @@
-
 function scrollUp() {
-    window.scrollTo({top:0, behavior: 'smooth'});
-}
\ No newline at end of file
+  window.scrollTo({top: 0, behavior: 'smooth'});
+}
+
+async function printComments() {
+  let limit = document.getElementById('limit').value;
```

Comment:
Suggest naming this (in the JS and HTML) something more descriptive - commentLimit maybe (comment-limit in the HTML)